addons/sale_project/models/project.py

```diff
@@ -823,22 +821,26 @@ def _group_expand_sales_order(self, sales_orders, domain, order):
             return search_on_comodel
         return sales_orders
 
-    @api.depends('sale_line_id', 'project_id', 'partner_id.commercial_partner_id', 'allow_billable')
+    @api.depends('sale_line_id', 'project_id.sale_order_id', 'partner_id.commercial_partner_id', 'allow_billable')
```

Comment:
I don't agree with that part: 
```txt
Also changes one of the dependent fields of the compute method from project_id to project_id.sale_order_id, assuming that when a task's sale_order_id is derived from its project's, changing the project's SO should trigger a recompute for the task's as well.
```
First, because we will change the behavior in stable and you do more than requested. 😕 
Then, the SO and/or the customer could be different between the project and the task, it would mean we will erase the SO of the task if the SO of the project changes, which is strange to me since the user could loose the materials linked to the task (industry_fsm_sale). 😕

Reply:
Fair point, undid the change.